streams/br.m3u

```diff
@@ -203,33 +203,15 @@ https://playplusrjo-lh.akamaihd.net/i/pp_rj@377859/index_720_av-p.m3u8
 https://playpluspoa-lh.akamaihd.net/i/pp_poa@377864/index_720_av-p.m3u8
 #EXTINF:-1 tvg-id="RecordTVSaoPaulo.br",Record TV SP (720p) [Geo-blocked]
 https://playplusspo-lh.akamaihd.net/i/pp_sp@350176/index_720_av-p.m3u8
-#EXTINF:-1 tvg-id="RedeBrasil.br",Rede Brasil
+#EXTINF:-1 tvg-id="RedeBrasil.br",Rede Brasil (432p)
 https://5cf4a2c2512a2.streamlock.net/rbtv/rbtv/.m3u8
 #EXTINF:-1 tvg-id="RedeBrasildeComunicacao.br",Rede Brasil de Comunicação (720p) [Not 24/7]
 http://rbc.directradios.com:1935/rbc/rbc/live.m3u8
 #EXTINF:-1 tvg-id="RedeCaxias.br",Rede Caxias (240p)
 https://stmv1.srvstm.com/luis5346/luis5346/playlist.m3u8
-#EXTINF:-1 tvg-id="RedeCNTCuiaba.br",Rede CNT Cuiaba (360p)
-https://d1s664t39qub1o.cloudfront.net/live/cnt-cuiaba_360p614k/index.m3u8
```

Comment:
Those links became tokened, have 3 different CDNs, all asking for a JWT token that can be found in a JS file : https://beyond.spalla.io/config/ee4d3ab8-a162-42d8-a4d0-e4ae03f9224a.js, the name of the js file being the videoID.